libsplinter/src/peer/mod.rs

```diff
@@ -213,7 +213,8 @@ impl PeerManager {
         retry_interval: u64,
         max_retry_attempts: u64,
         strict_ref_counts: bool,
-        identity: String,
+        // identity is not used if challenge-authorization is enabled
```

Comment:
Should this value be logged in the debug!() statement below?

Reply:
I added a commit to add it to the log